frontend/src/components/landing/PlanGrid.module.scss

```diff
@@ -0,0 +1,342 @@
+@use "~@mozilla-protocol/core/protocol/css/includes/lib" as *;
+@use "../../styles/color";
+@use "../../styles/text";
+
+.content {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  max-width: 1263px;
+  gap: $layout-md;
+}
+
+.header {
+  gap: $spacing-md;
+  text-align: center;
+
+  b {
+    font-family: Metropolis, sans-serif;
+    font-size: 28px;
+    font-style: normal;
+    line-height: 30px;
+  }
+
+  p {
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    padding-top: 8px;
+  }
+}
+
+.pricingPlans {
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-lg;
+
+  @media screen and (min-width: $screen-md) {
+    flex-wrap: wrap;
+    flex-direction: row-reverse;
+  }
+
+  @media screen and (min-width: $screen-lg) {
+    flex-wrap: nowrap;
+  }
+}
+
+.pricingCard {
+  box-shadow: inset 0 0 0 2px #e0e0e6;
+  background-color: $color-white;
+  border-radius: calc($border-radius-lg + $border-radius-md);
+  display: inline-flex;
+  flex: 1;
+  flex-direction: column;
+  font-size: 14px;
+  padding: $layout-sm $spacing-lg;
+  position: relative;
+  height: 645px;
+
+  &:first-of-type {
+    background: linear-gradient(
+      75deg,
+      $color-violet-50 0%,
+      $color-purple-50 37.1%,
+      $color-blue-40 61.4%,
+      #0090ed 100%
+    );
+    box-shadow: none;
+
+    @media screen and (min-width: $screen-md) {
+      min-width: 100%;
+    }
+
+    @media screen and (min-width: $screen-lg) {
+      min-width: auto;
+    }
+
+    &::before {
+      background: $color-white;
+      border-radius: calc($border-radius-lg + $border-radius-md - 2px);
+      content: "";
+      display: block;
+      height: calc(100% - 4px);
+      left: 2px;
+      position: absolute;
+      top: 2px;
+      width: calc(100% - 4px);
+    }
+  }
+
+  &Label {
+    color: $color-purple-70;
+    font-weight: 700;
+    left: 50%;
+    position: absolute;
+    top: $spacing-lg;
+    transform: translate(-50%);
+    white-space: nowrap;
+  }
+
+  dt,
+  dd {
+    // Ensure that the card content is not covered by the pseudo element
+    // `.pricingCard::before` that’s used for creating the border gradient of
+    // the bundle pricing card.
+    z-index: 1;
+  }
+
+  dt {
+    border-bottom: 1px solid #e0e0e6;
+    padding: $spacing-md 0;
+    text-align: center;
+
+    b {
+      font-size: 22px;
+      font-weight: 700;
+      padding: $spacing-xs 0;
+    }
+
+    p {
+      color: #3a3944;
```

Comment:
suggestion (blocking): This value is repeated 5 times. Make it a variable. And/or: can we re-use an existing color variable to keep consistency across the UI?